apps/src/blockly/googleBlocklyWrapper.js

```diff
@@ -187,6 +189,12 @@ function initializeBlocklyWrapper(blocklyInstance) {
     CdoRenderer,
     true /* opt_allowOverrides */
   );
+  blocklyWrapper.blockly_.registry.register(
```

Comment:
Preeety sure this shouldn't affect anything given we explicitly set the renderer elsewhere in this file, but maybe worth a quick check that this doesn't affect rendering of any of our google blockly labs (eg, poetry)?

Reply:
Glad you called this out, as it was a specific concern.  I've been manually checking https://studio.code.org/flappy/10 to make sure we don't change behavior in an existing Google Blockly level that uses the other renderer.